keylime-agent/src/agent_handler.rs

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2023 Keylime Authors
+
+use crate::common::JsonWrapper;
+use crate::{tpm, Error as KeylimeError, QuoteData};
+use actix_web::{web, HttpRequest, HttpResponse, Responder};
+use base64::{engine::general_purpose, Engine as _};
+use log::*;
+use serde::{Deserialize, Serialize};
+use std::{
+    fs::{read, read_to_string},
+    io::{Read, Seek},
+};
+
+#[derive(Serialize, Deserialize, Debug, Default)]
+pub(crate) struct AgentInfo {
+    pub agent_uuid: String,
+    pub tpm_hash_alg: String,
+    pub tpm_enc_alg: String,
+    pub tpm_sign_alg: String,
+    pub ak_handle: u32,
+}
+
+// This is a Info request which getting some information about this keylime agent
```

Comment:
I am not English native, but "which getting" sounds weird. Maybe "which is getting" or "which gets"?

Reply:
You're absolutely right, fixing now.